From 11722e850b667477caf4e678412de38ef1c3d574 Mon Sep 17 00:00:00 2001
From: penglezos <panagiotisegl@gmail.com>
Date: Mon, 16 May 2022 18:30:44 +0300
Subject: [PATCH] vendor: Build updater for all types

---
 config/version.mk | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/config/version.mk b/config/version.mk
index 5b71bd93..8fb51f39 100644
--- a/config/version.mk
+++ b/config/version.mk
@@ -33,10 +33,6 @@ ifeq ($(ARROW_OFFICIAL), true)
     ifeq ($(filter $(CURRENT_DEVICE), $(LIST)), $(CURRENT_DEVICE))
       IS_OFFICIAL=true
       ARROW_BUILD_TYPE := OFFICIAL
-
-PRODUCT_PACKAGES += \
-    Updater
-
     endif
     ifneq ($(IS_OFFICIAL), true)
        ARROW_BUILD_TYPE := UNOFFICIAL
@@ -56,6 +52,9 @@ ifeq ($(ARROW_COMMUNITY), true)
     endif
 endif
 
+PRODUCT_PACKAGES += \
+    Updater
+
 ARROW_VERSION := Arrow-$(ARROW_MOD_VERSION)-$(CURRENT_DEVICE)-$(ARROW_BUILD_TYPE)-$(shell date -u +%Y%m%d)-$(ARROW_BUILD_ZIP_TYPE)
 
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
